src/windows/main_window.py

```diff
@@ -2897,8 +2896,8 @@ def __init__(self, mode=None):
         self.show_property_type = None
         self.show_property_timer = QTimer()
         self.show_property_timer.setInterval(100)
```

Comment:
So, with setSingleShot(True), will the interval still work? For example, if 500 starts of this timer, will it keep canceling the current 100 ms timers? Or will it actually create 500 calls to `show_property_timeout`)... which is not want anyone wants of course.

Reply:
Each additional `start()` while the timer's already running will reset it back to 100ms, but leave it running. It will only issue a single `timeout` signal, after it's allowed to run for 100ms uninterrupted (un-reset, IOW) — at which point it'll automatically revert back to stopped state, until it's `start()`ed again.